libs/pika/executors/tests/unit/std_thread_scheduler.cpp

```diff
@@ -996,7 +1000,7 @@ void test_let_error()
             ex::let_error([=, &called](std::exception_ptr& ep) {
                 called = true;
                 check_exception_ptr_message(ep, "error");
-                return ex::transfer_just(sched);
+                return ex::schedule(sched);
```

Comment:
Because `just | continues_on` is overkill in this case. It's equivalent to the simpler `schedule` (it should've been just a `schedule` from the start).